metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -87,8 +87,8 @@ def step_init(obj, run_id, step_name, passed_in_split_indexes, task_id):
     "times to attach multiple tags.",
 )
 @click.option(
-    "--namespace",
-    "namespace",
+    "--kfp-namespace",
+    "kfp_namespace",
```

Comment:
Do you want to also provide a Metaflow `--namespace` ?

Reply:
yeah I was thinking we probably don't need it for now. but yeah let me add it back.

Reply:
Btw, just to be clear after discussing with metaflow folks about this parameter, it turns out this doesn't affect the write namespace (what tags are associated with these objects). It only affects the read namespace, meaning what objects are visible in the client API (I guess in case a run needs to access the artifacts of a different run in a different namespace!).

so didn't think this is needed now for any of the scenario. but anyway, will add it back just in case.

fyi, here is the param description:
  --namespace TEXT          Change namespace from the default (your username)
                            to the specified tag. Note that this option does
                            not alter tags assigned to the objects produced by
                            this run, just what existing objects are visible
                            in the client API. You can enable the global
                            namespace with an empty string.--namespace=